hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -42,9 +42,13 @@
   // Override the default vcs_cov_hier flag to supply chip specific coverage hierarchies.
   overrides: [
     {
-      name: vcs_cov_hier
+      name: default_vcs_cov_hier
       value: "-cm_hier {tool_srcs_dir}/chip_cover.cfg"
     }
+    {
+      name: cover_reg_top_vcs_cov_hier
+      value: "-cm_hier {tool_srcs_dir}/chip_cover_reg_top.cfg"
+    }
```

Comment:
Would it make sense to leave a comment for each of these overrides to explain what it's for?

Reply:
There's already a comment on line 42, but ok. I have added more detailed explanations.